lib/metatags/Metatags.js

```diff
@@ -1,73 +1,59 @@
 "use strict";
 
+var OpenGraphTag = require("./OpenGraphTag");
+var LinkTag = require("./LinkTag");
+var StandardTag = require("./StandardTag");
 var _ = require("underscore");
 
-var Metatags = function(pairs) {
-    if (!pairs) {
-        throw new Error("You cannot create empty metatags");
-    }
-
-    this.pairs = pairs;
-};
+var Metatags = {
 
-Metatags.prototype = {
+    tagViews: [],
 
-    pairs: null,
+    prevMetatags: null,
```

Comment:
this property for previous metatags may not work correctly for multiple users on the server side. it's probably safer to move this property to the Layout class so it's a part of a layout instance